app/controllers/site_controller.rb

```diff
@@ -395,6 +435,12 @@ def render_cluster
     end
     @axes = load_axis_labels
 
+    @axes_full = {
```

Comment:
Small style nit - we tend to use the JSON-style notation when using a `:symbol` for hash keys:

```
@axes_full = {
  titles: @axes,
  ranges: @range,
  aspects: @aspects
}
```

Reply:
Thanks for the note -- this is addressed in a related PR.